Modules/Accelerometer/mpu9250.py

```diff
@@ -122,14 +122,14 @@ def read_xyz(self, address, register, lsb):
 		Reads x, y, and z axes at once and turns them into a tuple.
 		"""
 		# data is MSB, LSB, MSB, LSB ...
-                # For some reason when we read 6 bytes we get (0,0,0,0,0,0)
-                # So we changed it to 7 bytes like in an arduino library we found
-                # And not it works, go figure.
-                data = self.bus.read_i2c_block_data(address, register, 7)
+		# For some reason when we read 6 bytes we get (0,0,0,0,0,0)
```

Comment:
You converted spaces to tabs on purpose??? Why not the other way around? :cry:

Reply:
The whole file used tabs

Reply:
:+1: